pkgs/development/tools/gdtoolkit/default.nix

```diff
@@ -24,7 +24,7 @@ let
 in
 python.pkgs.buildPythonApplication rec {
   pname = "gdtoolkit";
-  version = "3.3.1";
+  version = "4.2.2";
```

Comment:
You might also need to change the hash on line 34, otherwise it will fetch the old code. The easiest way is to replace the current hash with `lib.fakeSha256`and than build the package. Nix will then complain about the hash missmatch and give you the new hash.

Reply:
I added the fake sha, and when I try to build, I get the error: `error: gdtoolkit major version 4 does not match godot-server major version 3!`

The only mention of `godot-server` is an alias which will throw an error. Unsure how to move from here.

Reply:
This is caused by the assertion on line 65. I think it refers to `godot3-server` at the top. I did a quick serach and there seems to be no `godot4-server`. You might need to find out what exactly replaces `godot3-server`in version 4, and see if this is packaged alrady.